app/facades/npc_facade.rb

```diff
@@ -100,4 +100,15 @@ def self.create_npc(score_type, level)
     end
     npc
   end
+
+  def self.destroy_npc(npc_id)
+    NpcModel.where(id: npc_id).ids.map do |id|
+      NpcModelArmor.where(npc_model_id: id).destroy_all
+      NpcModelPack.where(npc_model_id: id).destroy_all
+      NpcModelWeapon.where(npc_model_id: id).destroy_all
+      NpcModelSpell.where(npc_model_id: id).destroy_all
+      UserNpcModel.where(npc_model_id: id).destroy_all
+    end
+    NpcModel.where(id: npc_id).destroy_all
```

Comment:
Because this method is being used in two places, I figured just put it in the `NpcFacade` so it can just be called on in the other two places